sdks/java/extensions/sql/zetasql/build.gradle

```diff
@@ -20,12 +20,18 @@ plugins {
   id 'org.apache.beam.module'
 }
 
+repositories {
```

Comment:
Will remove this when PR is ready to merge.

Reply:
Can't merge with this block.